Wikipedia/Code/TalkPageViewModel.swift

```diff
@@ -160,8 +173,13 @@ final class TalkPageViewModel {
             
             let activeUsersCount = activeUsersCount(topic: topic)
             
-            let topicViewModel = TalkPageCellViewModel(id: topic.id, topicTitle: topicTitle, timestamp: firstReply.timestamp, leadComment: leadCommentViewModel, replies: remainingCommentViewModels, activeUsersCount: activeUsersCount)
+            guard let topicName = topic.name else {
+                DDLogError("Unable to parse topic name")
+                return
```

Comment:
Let's do a `continue` here rather than `return`, so we try to turn the rest of the topics in the for loop into view models.